decorator3.py

```diff
@@ -0,0 +1,9 @@
+import decorator
+
+
+# calls the function it wraps twice
+@decorator.decorator
+def decorator(func, string):
+    func(string)
```

Comment:
This implementation suggests we use this decorator only for functions with string arguments. Use "args" instead.